VSRAD.Syntax/Core/TokenizerResult.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.VisualStudio.Text;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using VSRAD.Syntax.Core.Helper;
+using VSRAD.Syntax.Core.Tokens;
+
+namespace VSRAD.Syntax.Core
+{
+    public struct TokenizerResult : IEquatable<TokenizerResult>
```

Comment:
Does this need to be a struct? As far as I can tell, it's not used in arrays and only wraps reference types, so you don't get the memory savings/removal of indirection benefits from not using a class, and it's frequently passed as an argument, which creates copies.

If you think it should be a struct for readability reasons, please add the [`readonly`](https://devblogs.microsoft.com/premier-developer/avoiding-struct-and-readonly-reference-performance-pitfalls-with-errorprone-net/) modifier.

Reply:
Thank you, resolved: bd9d6df